# Questions per group
Q: [
  ['G1 Q1', 'G1 Q2'],
  ['G2 Q1', 'G2 Q2']
]

# Choices per question per group
C: [
  [
    ['G1 Q1 C1', 'G1 Q1 C2', 'G1 Q1 C3', 'G1 Q1 C4'],
    ['G1 Q2 C1', 'G1 Q2 C2', 'G1 Q2 C3', 'G1 Q2 C4']
  ],
  [
    ['G2 Q1 C1', 'G2 Q1 C2', 'G2 Q1 C3', 'G2 Q1 C4'],
    ['G2 Q2 C1', 'G2 Q2 C2', 'G2 Q2 C3', 'G2 Q2 C4']
  ]
]

# Correct answers (zero-based indexing)
A: [
  [0,'E2003412DC03011924274797'],
  ['E2003412DC03011924274797',3]
]

# RFID measurements file
rfid_file: 'results_001625143965.txt'
